src/utils/Dom.ts

```diff
@@ -481,7 +490,7 @@ export class Dom {
     } else {
       const modifiedType = this.processEventTypeToBeJQueryCompatible(type);
       const jq = JQueryUtils.getJQuery();
-      if (jq) {
+      if (jq && !Dom.forceNativeEvent) {
```

Comment:
I'd abstract that to a `shouldUseJQueryEvent()` method, and re-use it everywhere you need it.

Reply:
Seem like a good idea.